<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calc</class>
 <widget class="QDialog" name="Calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>633</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgba(255, 255, 255, 0);
	background-color: rgba(81, 255, 62,0);
  border: 3px rgb(31, 31, 31);
 border-radius:30px;
}
QPushButton:pressed {
    
	
	background-color: rgba(255, 255, 255, 111);
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>103</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12 9pt &quot;Okta Neue&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Ss/newcalc3.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="procent">
   <property name="geometry">
    <rect>
     <x>664</x>
     <y>103</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Okta Neue</family>
     <pointsize>9</pointsize>
     <weight>1</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12 9pt &quot;Okta Neue&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="years">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>320</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12 9pt &quot;Okta Neue&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month">
   <property name="geometry">
    <rect>
     <x>473</x>
     <y>320</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12 9pt &quot;Okta Neue&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="result">
   <property name="geometry">
    <rect>
     <x>827</x>
     <y>320</y>
     <width>131</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	background-color: rgba(25, 255, 255, 0);
	font: 75 20pt &quot;Century Gothic&quot;;
	color: rgba(255, 255, 255, 0);
   qproperty-alignment: 'AlignVCenter | AlignHCenter';
   border: 2px rgb(0, 0, 0);
border-radius:16px;
   

}

QPushButton:hover:!pressed
{
	
	background-color: rgba(45, 123, 196,55);
}


QPushButton:pressed {
    
	background-color: rgba(21, 255, 95, 20);
}

</string>
   </property>
   <property name="text">
    <string>Вычислить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>61</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 10pt &quot;Impact&quot;;
color:rgba(255, 170, 255,0);
	background-color: rgba(51, 51, 51, 0);
  border: 3px rgb(31, 31, 31);
 border-radius:30px;
}
QPushButton:hover:!pressed
{
	background-color: rgba(255, 133, 11, 55);
}


QPushButton:pressed {
    
	background-color: rgba(21, 255, 95, 20);
}

</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dot">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>141</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>836</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 10pt &quot;Impact&quot;;
color:rgba(255, 170, 255,0);
	background-color: rgba(51, 51, 51, 0);
  border: 3px rgb(31, 31, 31);
 border-radius:30px;
}
QPushButton:hover:!pressed
{
	background-color: rgba(255, 133, 11, 55);
}


QPushButton:pressed {
    
	background-color: rgba(21, 255, 95, 20);
}

</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valuta">
   <property name="geometry">
    <rect>
     <x>1077</x>
     <y>62</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>836</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 10pt &quot;Impact&quot;;
color:rgba(255, 170, 255,0);
	background-color: rgba(51, 51, 51, 0);
  border: 3px rgb(31, 31, 31);
 border-radius:30px;
}
QPushButton:hover:!pressed
{
	background-color: rgba(255, 133, 11, 55);
}


QPushButton:pressed {
    
	background-color: rgba(21, 255, 95, 20);
}

</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_plusminus">
   <property name="geometry">
    <rect>
     <x>1077</x>
     <y>140</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 10pt &quot;Impact&quot;;
color:rgba(255, 170, 255,0);
	background-color: rgba(51, 51, 51, 0);
  border: 3px rgb(31, 31, 31);
 border-radius:30px;
}
QPushButton:hover:!pressed
{
	background-color: rgba(255, 133, 11, 55);
}


QPushButton:pressed {
    
	background-color: rgba(21, 255, 95, 20);
}

</string>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255,0);
border:0px;
font: 75 22pt  &quot;Okta Neue&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 80);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="pick">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>173</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>381</width>
     <height>31</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>219</red>
         <green>219</green>
         <blue>219</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>219</red>
         <green>219</green>
         <blue>219</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>219</red>
         <green>219</green>
         <blue>219</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>219</red>
         <green>219</green>
         <blue>219</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>219</red>
         <green>219</green>
         <blue>219</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>219</red>
         <green>219</green>
         <blue>219</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>219</red>
         <green>219</green>
         <blue>219</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>219</red>
         <green>219</green>
         <blue>219</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>219</red>
         <green>219</green>
         <blue>219</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	background-color: rgb(219, 219, 219);
border-radius:10px;
	color: rgb(0, 0, 0);
}

QComboBox::Activated{
background-color:rgb(121,71,71);
}</string>
   </property>
   <property name="maxVisibleItems">
    <number>39</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1. Вклад№1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2. Вклад№2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3. Вклад№3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4. Вклад№4</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_info">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>173</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgba(255, 255, 255, 0);
	background-color: rgba(81, 255, 62,0);
  border: 3px rgb(31, 31, 31);
 border-radius:15px;
}
QPushButton:pressed {
    
	
	background-color: rgba(255, 255, 255, 111);
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dark">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>570</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{background-color: rgba(255, 255, 255, 111);
background-image: url(:/Ss/switchnight.png);
border-radius: 55px;
}
QPushButton:hover
{
	background-color: rgba(128, 214, 250,22);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="days">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>320</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12 9pt &quot;Okta Neue&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="rub">
   <property name="geometry">
    <rect>
     <x>468</x>
     <y>77</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_plusminus</zorder>
  <zorder>pick</zorder>
  <zorder>procent</zorder>
  <zorder>pushButton_dot</zorder>
  <zorder>value</zorder>
  <zorder>month</zorder>
  <zorder>textBrowser</zorder>
  <zorder>result</zorder>
  <zorder>years</zorder>
  <zorder>pushButton_info</zorder>
  <zorder>valuta</zorder>
  <zorder>clear</zorder>
  <zorder>pushButton</zorder>
  <zorder>dark</zorder>
  <zorder>days</zorder>
  <zorder>rub</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
